lisa/sut_orchestrator/azure/common.py

```diff
@@ -511,6 +511,8 @@ def get_compute_client(
         credential=platform.credential,
         subscription_id=subscription_id,
         api_version=api_version,
+        base_url=platform.cloud.endpoints.resource_manager,
```

Comment:
For each part, it can pass in the whole cloud object. So it's flexible to take parts from it.

Reply:
only resource manager is used in this method

Reply:
I mean for every place, pass the object in. That will get all logic consistent, and simple to manage the interface.

Reply:
I think passing platform would be enough.
```
def get_compute_client(
    platform: "AzurePlatform",
    api_version: Optional[str] = None,
    subscription_id: str = "",
) -> ComputeManagementClient:
```

Reply:
All other places should be fixed to pass cloud object, not only one or two used endpoints.

Reply:
updated